src/bundle/Security/TwoFactor/Event/Event.php

```diff
@@ -2,19 +2,19 @@
 
 declare(strict_types=1);
 
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Christian Scheb
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
 namespace Scheb\TwoFactorBundle\Security\TwoFactor\Event;
 
-use Symfony\Component\EventDispatcher\Event as BaseEvent;
-use Symfony\Component\EventDispatcher\EventDispatcher;
 use Symfony\Contracts\EventDispatcher\Event as ContractsBaseEvent;
-use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 
-if (is_subclass_of(EventDispatcher::class, EventDispatcherInterface::class)) {
-    class Event extends ContractsBaseEvent
-    {
-    }
-} else {
-    class Event extends BaseEvent
-    {
-    }
+class Event extends ContractsBaseEvent
```

Comment:
Missed this when cleaning up for 4.4, will take it over as a clean-up change.

Reply:
Added to https://github.com/scheb/2fa/commit/dd1d48194488b009ff482674021350b33d1487c6